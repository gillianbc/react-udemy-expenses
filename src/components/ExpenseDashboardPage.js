// Stateless functional component
import React from "react";
import MyExpenses from "./ExpenseList";

const ExpenseDashboardPage = () => (
    <div>
        <MyExpenses />
    </div> )

export default ExpenseDashboardPage