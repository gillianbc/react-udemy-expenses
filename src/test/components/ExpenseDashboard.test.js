import React from 'react'
import { shallow } from 'enzyme'
import ExpenseDashboardPage from "../../components/ExpenseDashboardPage";
/*
* Tests the snapshot of the NotFoundPage component
* */

it('The snapshot should match when there is expense data', () => {
    // Remember, these curlies are not object brackets, they're JSX brackets
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
})



