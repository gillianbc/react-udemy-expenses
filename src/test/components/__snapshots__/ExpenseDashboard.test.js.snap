// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The snapshot should match when there is expense data 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
